ws/src/speech/scripts/Whisper.py

```diff
@@ -83,32 +82,41 @@ def interpret(self, data):
         WavUtils.discard_wav(temp_file)
 
         if not empty:
-            return result["text"] 
-        
+            return result["text"]
+
+    def infer_wav(self, wav_path):
+        result = self.audio_model.transcribe(
+            wav_path, fp16=torch.cuda.is_available())
+        return result["text"]
```

Comment:
unused

Reply:
sorry used but not here, please clarify this with a comment that this is a test function or even name it differently

Reply:
Got it. Instead I refactored to use also within file on infer method.